plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/credentials/pinning/QConnection.kt

```diff
@@ -6,13 +6,19 @@ package software.aws.toolkits.jetbrains.core.credentials.pinning
 import software.aws.toolkits.jetbrains.core.credentials.AwsBearerTokenConnection
 import software.aws.toolkits.jetbrains.core.credentials.ToolkitConnection
 import software.aws.toolkits.jetbrains.core.credentials.sono.Q_SCOPES
+import software.aws.toolkits.jetbrains.core.credentials.sono.Q_SCOPES_UNAVAILABLE_BUILDER_ID
```

Comment:
probably need to change the scopes we're logging in with

https://github.com/aws/aws-toolkit-jetbrains/pull/4214
https://github.com/aws/aws-toolkit-jetbrains/pull/4300